.test_runner_config.yaml

```diff
@@ -40,7 +40,8 @@ steps:
     -a ${server_password} --setup-dns --auto-forwarders
   - ipa-kra-install -p ${server_password}
   lint:
-  - make V=0 lint
+  - PYTHON=/usr/bin/python2 make V=0 lint
+  - PYTHON=/usr/bin/python3 make V=0 lint
```

Comment:
please make the python3 target ```pylint``` only. I just realized that lint also runs the i18n and JS linters.